ssr/bundle-runner.js

```diff
@@ -40,7 +44,22 @@ module.exports.compileModule = filepath => {
 		//
 		// So we call the function, passin the exports as the "this" object,
 		// then pass in the appropriate other stuff.
-		compiledWrapper.call(m.exports, m.exports, require, m);
+		compiledWrapper.call(
+			m.exports,
+			m.exports,
+			// This is the "require" function within the script. We hijack it.
+			id => {
+				// If it's a relateive path, resolve it relative to the entry
+				// file.
+				if (id.startsWith('.')) {
+					id = path.resolve(path.dirname(filepath), id);
+				}
+				return require(id);
+			},
+			m,
+			filepath,
+			path.dirname(filepath)
+		);
```

Comment:
comment on why the require function has problems with relative paths in the first place and if theres some documentation you can point at for this. i don't fully understand why this fixes ssr.